incuna-sass/mixins/_svg.sass

```diff
@@ -0,0 +1,12 @@
+//////////  S V G  M I X I N  //////////////////////////////
+// To use like you would a sprite with the added benefit that
+// it can be whatever size you like and will work perfectly
+// on retina screens
+@mixin svg($file, $width: 100%, $height: $width)
+    @include background(image-url('svg/#{$file}.svg'))
+    background:
+        repeat: no-repeat
+        size: contain
+    width: $width
+    height: $height
```

Comment:
I think `width` and `height` would be more suited to a separate mixin called `svg-element` or `svg-sprite`. It would be nice to have a `@mixin svg` that you can use to conveniently set the background image (with `background-size` nicely containing the image) without forcing dimensions on it.

But, I don't see much of a convenience to being able to pass `width` and `height` into a mixin when all it does is set those properties: we can do that ourselves after using `@include svg`. There is a slight convenience to passing only one value and getting a square, but I think it would read better as explicit `width` and `height` styles local to the `@include svg` call.